src/components/Element.test.js

```diff
@@ -100,7 +100,7 @@ describe('Element', () => {
     });
   });
 
-  it.only("re-rendering with new props should still work if addElementsLoadListener hasn't fired yet", () => {
+  it("re-rendering with new props should still work if addElementsLoadListener hasn't fired yet", () => {
```

Comment:
Is it worth doing the work to fail a build if any of these focus test methods are used?

Reply:
I thought the same thing! we should investigate if we can make jest fail when there are skipped tests.